packages/grafana-ui/src/options/builder/tooltip.tsx

```diff
@@ -52,6 +52,7 @@ export function addTooltipOptions<T extends OptionsWithTooltip>(
         integer: true,
         placeholder: '300',
       },
+      showIf: (options: T) => options.tooltip?.mode !== TooltipDisplayMode.None,
```

Comment:
```suggestion
      showIf: (options: T) => config.featureToggles.newVizTooltips && options.tooltip?.mode !== TooltipDisplayMode.None,
```

Reply:
hm just tried to add this locally and ran into issue where it seems we are not able to pull the `config` object from grafana runtime anymore in the `grafana-ui` package... made this thread in [frontend channel](https://raintank-corp.slack.com/archives/CHKLEJ668/p1706741905259889) to ask about this 😬

Reply:
ugh, yeah, I forgot about imports 😬  it's not possible as far as I know, I've also asked a while ago and we ended up moving Timeseries out of `grafana-ui` and do the imports.. So maybe we should leave it as is, we're going to remove the feature flag in the near future anyway 😞

Reply:
Following up from thread it appears there is no way to access the feature toggles from the run time config

The only way that this would be feasible is through passing this info as a prop to the component - which could be a viable solution for the time being

Reply:
We may need to talk about this as currently these options are being exposed regardless of the tooltip usage. This PR doesn't impact that so I may go ahead and merge anyhow, and we can address this being displayed in another PR